idl/streaming.idl.hh

```diff
@@ -12,6 +12,10 @@
 
 namespace streaming {
 
+class plan_id {
+    utils::UUID uuid();
+};
```

Comment:
I think this needs to be `final` or IDL will encode the struct size and break compatibility.

Reply:
Fixed in 314e45d957